docs/getting-started-with-scalardb.md

```diff
@@ -138,6 +139,7 @@ public class ElectronicMoneyWithStorage extends ElectronicMoney {
 ```
 
 Now we can run the application.
+If you run those command in the getting-started repository do not forget to create an empty file named `setting.gradle`.
```

Comment:
I found this issue last Friday, and I added `setting.gradle` in #233. So I think we can remove this line. Thanks.

Reply:
Yes, Above sentence is not needed because `setting.gradle` available in the repository
It is working fine.